llvm/lib/Transforms/PeepholeOptimizationCourse/Functions/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_llvm_component_library(
+    LLVMFunctions
```

Comment:
I don't see where you link against that library - how did linker find implementations?